packages/ses-ava/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/ses-ava",
-  "version": "0.1.1+1-dev",
+  "version": "0.2.0",
```

Comment:
This should be v0.1.2. Existing dependees should be able to pick up the upgrade with `^0.1` since there are no breaking changes.

Reply:
You added a feature.